Parsers/Animal Facts.js

```diff
@@ -0,0 +1,67 @@
+/*
+activation_example:!animal name of animal
+regex:!animal
+flags:gmi
+*/
+
+var where = current.text.indexOf('!animal ') + 8;
+var term = current.text.substr(where).trim();
+var chatReq = new sn_ws.RESTMessageV2();
+chatReq.setEndpoint('https://api.api-ninjas.com/v1/animals?name=' + term);
+chatReq.setHttpMethod("GET");
+chatReq.setRequestHeader('X-Api-Key', 'tbfgkwW8c0UY0xE0Yd9oSw==DusJx995ZmqecaPW');
```

Comment:
Merging would share your key more broadly. I'd suggest putting this into a global property and then requesting that it be added by the team